includes/admin/settings/class-wc-settings-emails.php

```diff
@@ -109,7 +109,7 @@ public function get_settings() {
 				'placeholder' => __( 'N/A', 'woocommerce' ),
 				'type'        => 'textarea',
 				/* translators: %s: site name */
-				'default'     => sprintf( __( '%s - Powered by WooCommerce', 'woocommerce' ), get_bloginfo( 'name', 'display' ) ),
+				'default'     => sprintf( __( '%s', 'woocommerce' ), get_bloginfo( 'name', 'display' ) ),
```

Comment:
@jameskoster There is no translateable text here :)

Reply:
haha, you're right.